src/MatBlazor.Demo/Demo/DemoMatSlider.razor

```diff
@@ -283,3 +283,51 @@
     ")></BlazorFiddle>
     </SourceContent>
 </DemoContainer>
+
+<h5 class="mat-h5">Value update</h5>
+<DemoContainer>
+    <Content>
+        <MatTextField @bind-Value="@Value" Label="Value"></MatTextField>
+        <MatTextField @bind-Value="@ValueMin" Label="ValueMin"></MatTextField>
+        <MatTextField @bind-Value="@ValueMax" Label="ValueMax"></MatTextField>
+        <MatSlider TValue="decimal" Value="@Value" ValueMin="@ValueMin" ValueMax="@ValueMax" ValueChanged="@OnValueChanged2"></MatSlider>
+
+        @code
+        {
+
```

Comment:
This extra line isn't consistent with the rest of the code standards

Reply:
i think it is consistent with this actual file. in all the other examples there are all of the extra lines mentioned in this comments. i agree that it could be fixed according to code standards but then it should be whole file, not only this particular example

Reply:
This is quite trivial but understandably needed. I see several places in MatBlazor with and without and line after `@code`  This again could / should be address in a PR for MatBlazor as a whole.